lib/static/new-ui/features/suites/components/TreeViewItemSubtitle/index.tsx

```diff
@@ -12,11 +16,23 @@ interface TreeViewItemSubtitleProps {
 }
 
 export function TreeViewItemSubtitle(props: TreeViewItemSubtitleProps): ReactNode {
-    if (props.item.type === TreeViewItemType.Browser && props.item.diffImg) {
-        return <ImageWithMagnifier src={props.item.diffImg.path} alt={'diff-image'} style={{maxWidth: '99%', marginTop: '4px', maxHeight: '40vh'}} scrollContainerRef={props.scrollContainerRef} />;
+    if (props.item.type === TreeViewItemType.Browser && props.item.images?.length) {
+        return <div>
+            {props.item.images.map((imageEntity, index) => {
+                const image = (imageEntity as ImageEntityFail).diffImg ?? (imageEntity as ImageEntityFail).actualImg;
+                if (!image) {
+                    return;
+                }
+
+                return <div key={index}>
+                    <span className={styles.imageStatus}>{(imageEntity as ImageEntityFail).stateName} ⋅ {getAssertViewStatusMessage(imageEntity)}</span>
+                    <ImageWithMagnifier image={image} style={{maxWidth: '99%', marginTop: '4px', maxHeight: '40vh'}} scrollContainerRef={props.scrollContainerRef}/>
+                </div>;
+            })}
+        </div>;
     } else if (props.item.type === TreeViewItemType.Browser && props.item.errorStack) {
         return <div className={classNames(styles['tree-view-item-subtitle__error-stack'], props.className)}>
-            {props.item.errorStack}
+            {stripAnsi(props.item.errorStack)}
```

Comment:
What is item.errorStack?
What is this component?

Reply:
TreeViewItem — component that's used for displaying nodes of the tree view. In Gravity UI, it can accept subtitle. We use subtitle to render previews of frequently needed info, e.g. error of last retry or diff of last retry.

`errorStack` is used to store error stack/snipped of last retry. I'll probably add comments to the item interface in the next PR.